.github/workflows/upload_component.yml

```diff
@@ -72,6 +72,7 @@ jobs:
             usb/usb_host_vcp;
             zlib;
             libpng;
+            ccronexpr;
```

Comment:
Since we are wrapping the upstream library and not adding any new functionality, would it make sense to use the original name (`supertinycron`)?